### DISCUSSION TITLE
How to disable specific heading levels in the Editor

### DISCUSSION THREAD

**Discussion by ndiego** [WordPress/developer-blog-content#323] [Topic Ideas]
This would be a snippet that provides an example of the heading-level curation options available in WordPress 6.7. 

The snippet will show you how to disable H1, H5, and H6 in the Heading block for all users who are not administrators using PHP. 

```php
function example_modify_heading_levels_globally( $args, $block_type ) {

    $is_administrator = current_user_can( 'edit_theme_options' );

    if ( 'core/heading' !== $block_type || $is_administrator ) {
        return $args;
    }

    // Remove H1, H5, and H6.
    $args['attributes']['levelOptions']['default'] = [ 2, 3, 4 ];

    return $args;
}
add_filter( 'register_block_type_args', 'example_modify_heading_levels_globally', 10, 2 );
```

**Comment by ndiego** [comment]
While this snippet is not yet approved, I have drafted it if anyone would like to provide a preliminary review: https://docs.google.com/document/d/1BOuO8eqqMfVae-wBzfItBOr5MMuuO0heY7_kJG07_OM/edit

**Comment by bph** [comment]
The Snippet is approved and conversation continues on the issue: 
- #338

**Comment by ndiego** [comment]
The post (snippet) has been published here: https://developer.wordpress.org/news/snippets/how-to-disable-heading-levels-in-the-editor/
